### PR TITLE
elementary-planner: 2.6.9 -> 2.7

### PR DESCRIPTION
**Pull request by k0ral**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/office/elementary-planner/default.nix [hunk 1]
```diff
@@ -58,6 +60,8 @@ stdenv.mkDerivation rec {
     sqlite
     webkitgtk
     libgdata # required by some dependency transitively
```
Comment (SuperSandro2000) [inline pkgs/applications/office/elementary-planner/default.nix:62]: You don't know which exactly?

Comment (k0ral) [reply pkgs/applications/office/elementary-planner/default.nix:62]: I'm not sure what's the rationale for this comment, as a matter of fact `libgdata` is listed as a direct dependency in [`meson.build`](https://github.com/alainm23/planner/blob/master/meson.build).

**Comment by vojta001** [comment]
I can confirm it can be built and run. The window looks horrible with tiling WM, but that might not be a result of this change

**Review by bobby285271 — APPROVED** [review]
Thanks for your contribution!

Change looks good to me, builds.

- https://github.com/alainm23/planner/releases/tag/2.7
- https://github.com/alainm23/planner/compare/2.6.9...2.7

> - New Natural Language Date Parser.
> - First beta support for CalDAV.
> - Labels synced with Todoist.
> - Many, many design improvements.

Tested some of the basic functionalities, also seems good.

**Pull request merged**